jujugui/static/gui/src/app/components/profile/navigation/navigation.js

```diff
@@ -0,0 +1,62 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+
+'use strict';
+
+class ProfileNavigation extends React.Component {
```

Comment:
Docstring for the component, how is it used, for what purpose, what does it provide?